qemu/tests/rng_bat.py

```diff
@@ -129,6 +143,13 @@ def get_driver_id(session, cmd, pattern):
                 raise exceptions.TestFail("Unable to read random numbers from"
                                           "guest: %s" % output)
     else:
+        if host_read_cmd:
+            host_read_process = process.SubProcess(host_read_cmd)
+            host_read_process.start()
+
+            if not check_host_read_running(host_read_cmd):
+                raise exceptions.TestFail("Host reading data is not alive!")
```

Comment:
Please use `test.fail()` instead, but my understand here is it is more like an error

Reply:
Modified,thanks